### ISSUE TITLE
Version 2.0.0 not building with use_frameworks

### ISSUE THREAD

**Issue by Almouro** [getsentry/sentry-react-native#1171]
**OS:**
- [ ]  Windows
- [x]  MacOS
- [ ]  Linux

**Platform:**
- [x]  iOS
- [ ]  Android

**SDK:**
- [x]  `@sentry/react-native` (>= 1.0.0)
- [ ]  `react-native-sentry` (<= 0.43.2)

**SDK version:** 2.0.0

**`react-native` version:** 0.63.3

**Are you using Expo?**
- [ ]  Yes
- [x]  No

**Are you using sentry.io or on-premise?**
- [x]  sentry.io (SaaS)
- [ ]  on-premise

**If you are using sentry.io, please post a link to your issue so we can take a look:**

*Not relevant*

**Configuration:**

*Not relevant*

---
**I have following issue:**

When using `use_frameworks!` in the Podfile, I can't build RNSentry and get the following error:

**Steps to reproduce:**
- `react-native init TestSentry`
- Add `use_frameworks!` in Podfile (which produces `The 'Pods-TestSentry' target has transitive dependencies that include statically linked binaries` error when running `pod install`)
- Add this workardound in the Podfile to make `pod install` work:
```ruby
pre_install do |installer|
  # See https://github.com/CocoaPods/CocoaPods/issues/3289
  Pod::Installer::Xcode::TargetValidator.send(
    :define_method,
    :verify_no_static_framework_transitive_dependencies
  ) {}
end
```

**Actual result:**

Getting this error: `'Sentry/SentryFileManager.h' file not found` when building the app

Perhaps should the file be added into this folder on `sentry-cocoa`? https://github.com/getsentry/sentry-cocoa/tree/master/Sources/Sentry/Public

Previous version of the SDK is building fine however

**Expected result:**

The app should bulld

**Comment by jennmueng** [comment]
The fix has been introduced in this PR to `@sentry/cocoa` https://github.com/getsentry/sentry-cocoa/pull/835

**Comment by hanayashiki** [comment]
Looking forward to the update!

**Comment by jennmueng** [comment]
This issue has been fixed in version [2.0.1](https://github.com/getsentry/sentry-react-native/releases/tag/2.0.1). Please upgrade and don't forget to run `pod install`!

**Comment by Almouro** [comment]
Thanks @jennmueng ! Works well now 👍
